chapters/sampling-randomization-power.tex

```diff
@@ -50,38 +50,33 @@ \section{Random processes in Stata}
 and treatment assignment processes are truly random.
 Therefore, understanding and correctly programming for sampling and randomization
 is essential to ensuring that planned experiments
-are correctly implemented in the field, so that the results
+are correctly implemented, so that the results
 can be interpreted according to the experimental design.
 There are two distinct random processes referred to here:
 the conceptual process of assigning units to treatment arms,
 and the technical process of assigning random numbers in statistical software,
 which is a part of all tasks that include a random component.\sidenote{
   \url{https://blog.stata.com/2016/03/10/how-to-generate-random-numbers-in-stata}}
 
+Any process that includes a random component is a random process,
+including sampling, randomization, power calculation simulations, and algorithms like bootstrapping.
 Randomization is challenging and its mechanics are unintuitive for the human brain.
 ``True'' randomization is also nearly impossible to achieve for computers,
 which are inherently deterministic.\sidenote{
   \url{https://www.random.org/randomness}}
+
+\subsection{Implementing random processes reproducibly in Stata}
+
+Reproducibility in statistical programming means that the outputs of random processes
+can be re-obtained at a future time.\cite{orozco2018make}
```

Comment:
```suggestion
to be re-obtained at a future time.\cite{orozco2018make}
```

The way it was phrased before made it seem to me that reproducing just the random processes is enough to be considered reproducible. But we can keep it as was if I was the only one who thought so, or if this is not clearer

Reply:
I accepted the first suggestion but I don't understand the second. With the second suggestion, the sentence would read "Reproducibility in statistical programming means that the outputs of random processes **to** be re-obtained at a future time". that's not grammatically correct. i think **can** still makes sense here but let me know if i'm missing something.